src/Containers/FileUpload/FileUpload.stories.tsx

```diff
@@ -13,15 +13,50 @@ export default {
         setBase64: (base64StringFile: string) => {
             console.log(base64StringFile);
         },
-        isUploading: false,
-        isSuccess: false,
-        isFailure: false,
         successMessage: 'Completed',
         failureMessage: 'Something went wrong',
         disabled: false,
     },
 } as Meta;
 
-export const Basic: Story<IFileUploadProps> = (args) => (
-    <FileUpload {...args} />
-);
+export const Basic: Story<IFileUploadProps> = (args) => {
+    const [isUploading,setIsUploading]=useState(false)
+    const [isSuccess,setIsSuccess]=useState(false)
+    const [isFailure,setIsFailure]=useState(false)
+
+    const componentProps={
+        isUploading,isSuccess,isFailure,setIsUploading,setIsSuccess,setIsFailure
+    }
+
+    return (
+        <FileUpload {...args} {...componentProps} />
+    )
+};
+
+export const VeryLongMessageDuration: Story<IFileUploadProps> = (args) => {
```

Comment:
Why duplicate entire code? Why not just pass duration as a prop?

Reply:
I have done it like this because passing it as a prop in args doesn't work with storybook (there is no difference when changing the value in storybook controls) . so putting it this way, with different stories, it's how you see the difference that this value make on the behaviour of the component

Reply:
messageDuration it's a prop, it's an optional prop

Reply:
It does work in storybook, you need to define subComponentProps to access in storybook